mesonbuild/compilers/rust.py

```diff
@@ -121,7 +121,7 @@ def get_optimization_args(self, optimization_level: str) -> T.List[str]:
         return rust_optimization_args[optimization_level]
 
     def compute_parameters_with_absolute_paths(self, parameter_list: T.List[str],
-                                               build_dir: str) -> T.List[str]:
+                                               build_dir: str) -> T.List[str]:#
```

Comment:
No idea what this is, probably got accidentally committed.

Reply:
typo indeed, nice catch. Fixed.